backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/proto/CreateShopRequest.java

```diff
@@ -0,0 +1,12 @@
+package com.google.gpay.virtualqueue.backendservice.proto;
```

Comment:
Apache license info missing. Please check if that's required here.

Reply:
added!